lib/routes/gdufs/news.ts

```diff
@@ -0,0 +1,89 @@
+import { Route } from '@/types';
+import { load } from 'cheerio';
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+
+const site = 'https://www.gdufs.edu.cn';
+
+export const route: Route = {
+    path: '/news',
+    categories: ['university'],
+    example: '/gdufs/news',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['https://www.gdufs.edu.cn/gwxw/gwxw1.htm', 'https://www.gdufs.edu.cn/'],
```

Comment:
```suggestion
            source: ['www.gdufs.edu.cn/gwxw/gwxw1.htm', 'www.gdufs.edu.cn/'],
```
Do not include protocel in `source`